soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2944,6 +2946,9 @@ void func_80835EFC(Player* this) {
     }
 }
 
+// used for tunic buffers
+u8 BufferTunicsNextFrame = false;
```

Comment:
This flag should live somewhere in SoH code (soh/Enhancements/?), where you can also define an extern "C" function like "setRefreshTunicFlag()". Then wherever you call that function, put it under some CVar conditional so that it's clearly separated from original game code. Maintainers should be able to help with exactly where it should go.

Reply:
I'm not exactly sure how to do that

Reply:
Well, I've set it up as an extern function, but I don't really like having to set stuff like this up as a CVar that would have to be toggled in enhancements or something because then enabling the model ends up being less seamless, and I'm not the personal biggest fan of that